docs/reference/overrides/select-value.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "SelectValue",
+  "props": {
+    "children": {
+      "type": "React.ReactElement | (value) => React.ReactElement",
+      "description": "Allows you to customize the rendering of the text.\n\nAccepts a `ReactElement` or a function that returns the element to render."
```

Comment:
@vladmoroz If you're ok with this description I'll use the same in https://github.com/mui/base-ui/pull/990

Reply:
I’d say:

```
Override children if you need to customize how the value is displayed.
By default, renders the current value as is.

Accepts a `ReactElement` or a function that returns the element to render.
```

***

Also, shouldn't the children render function allow `React.ReactNode` instead of `React.ReactElement`?